Makefile

```diff
@@ -5,6 +5,7 @@ BUILDTAGS=seccomp
 export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
 
 all:
+	ln -sfn $(CURDIR) $(CURDIR)/Godeps/_workspace/src/github.com/opencontainers/runc
 	go build -tags "$(BUILDTAGS)" -o runc .
```

Comment:
I'm not sure about godeps, but if it causes any issues, we can just use another directory instead (and add it to GOPATH). Something like: https://github.com/gitido/runc/commit/4539dee507473ccf8dd112ecfb0148f9e80e1f0d.

But I'm OK with your patch as well :)